python/lsst/ts/standardscripts/base_focus_sweep.py

```diff
@@ -120,6 +120,12 @@ def get_schema(cls) -> dict:
                 description: Number of steps to take inside the focus window.
                 type: number
                 minimum: 3
+              focus_steps:
```

Comment:
Maybe we could use a more descriptive name and description here? As-is, it kind of looks like it is something we would use with focus_window and n_steps. Maybe instead of `focus_steps`, this could be `focus_step_sequence`?